website/docs/add-secure-apps/flows-stages/flow/context/index.md

```diff
@@ -189,3 +195,11 @@ Optionally override the email address that the email will be sent to. If not set
 ##### `pending_user_identifier` (string)
 
 If _Show matched user_ is disabled, this key will be set to the user identifier entered by the user in the identification stage.
+
+#### Redirect stage
+
+##### `redirect_stage_target` (string)
+
+[Set this key](../../../../customize/policies/expression/managing_flow_context_keys.md) in an Expression Policy to override [Redirect stage](../../stages/redirect/index.md) to force it to redirect to a certain URL or flow. This is useful when a flow requires that the redirection target be decided dynamically.
```

Comment:
Yesterday this sentence made perfect sense to me, lol. But tonight I am unsure of this part: "...an Expression Policy to override [Redirect stage](../../stages/redirect/index.md) to force it to redirect to a certain URL or flow."

Are we **overriding** the Redirect stage, or simply utilizing the Redirect stage? The override part is a bit confusing to me. Oh, wait, would an Admin use this Expression policy to target specific users or groups?? Like, all these people get the regular redirect stage, which will tkae them where they need to go. But THESE people, identified in the Expression, should be redirected to this other place...??

Reply:
Setting this key overrides the Redirect stage's `mode` and either `target_static` or `target_flow` (whichever fits the new `mode`). I think "override the Redirect stage" is a good enough wording of that, but I'm happy to adjust it.

---

> would an Admin use this Expression policy to target specific users or groups?

Correct. Let's say you have a Redirect stage with `mode == 'flow'` and `target_flow.slug == 'default-flow'`. Then you can do something like

```
if user_fits_criterion_1:
    context['flow_plan'].context['redirect_stage_target'] = 'ak-flow://custom-flow-1'
elif user_fits_criterion_2:
    context['flow_plan'].context['redirect_stage_target'] = 'ak-flow://custom-flow-2'

# all other users will be redirected according to the Redirect stage's settings
# in this case, to `default-flow`
```